executor/simple.go

```diff
@@ -66,6 +68,8 @@ var (
 	transactionDurationOptimisticRollback  = metrics.TransactionDuration.WithLabelValues(metrics.LblOptimistic, metrics.LblRollback)
 )
 
+const notSpecified = -1
```

Comment:
unspecified?

Reply:
The variable is not used in this DDL statement